operator/src/main/resources/scripts/utils.sh

```diff
@@ -17,6 +17,27 @@ source ${SCRIPTPATH}/utils_base.sh
 #   [ $? -ne 0 ] && echo "[SEVERE] Missing file ${SCRIPTPATH}/utils.sh" && exit 1
 #
 
+#
+# Define helper fn to copy a file only if src & tgt differ
+#
+
+function copyIfChanged() {
+  [ ! -f "${1?}" ] && trace SEVERE "File '$1' not found." && exit 1
+  if [ ! -f "${2?}" ] || [ ! -z "`diff $1 $2 2>&1`" ]; then
+    trace "Copying '$1' to '$2'."
+    tmp_file=$(mktemp -p $(dirname $2))
+    cp $1 $tmp_file
+    mv $tmp_file $2
```

Comment:
Copying to a tmp file and then moving the file is a little subtle. Perhaps add a comment about why we're doing this?